x/ccv/child/keeper/distribution.go

```diff
@@ -0,0 +1,114 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	transfertypes "github.com/cosmos/ibc-go/v3/modules/apps/transfer/types"
+	clienttypes "github.com/cosmos/ibc-go/v3/modules/core/02-client/types"
+	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+
+	"github.com/cosmos/interchain-security/x/ccv/child/types"
+	ccv "github.com/cosmos/interchain-security/x/ccv/types"
+)
+
+// Simple model, send tokens to the fee pool of the provider validator set
+// reference: cosmos/ibc-go/v3/modules/apps/transfer/keeper/msg_server.go
+func (k Keeper) DistributeToProviderValidatorSet(ctx sdk.Context) error {
+
+	ltbh, err := k.GetLastTransmissionBlockHeight(ctx)
+	if err != nil {
+		return err
+	}
+	bpdt := k.GetBlocksPerDistributionTransmission(ctx)
+	curHeight := ctx.BlockHeight()
+
+	if (curHeight - ltbh.Height) < bpdt {
+		// not enough blocks have passed for  a transmission to occur
+		return nil
+	}
+
+	// work around to reuse the IBC token transfer logic
```

Comment:
There actually is no real work around, so I just removed this comment. We just need module the address (next line), normally with module accounts you just send the string name of the account not the address, but because ibc-token transfer was designed for non-module accounts it expects the address... but it's simple to determine, so I wouldn't really even call this a workaround.